tools/crm_node.c

```diff
@@ -130,6 +130,16 @@ remove_cb(const gchar *option_name, const gchar *optarg, gpointer data, GError *
     return TRUE;
 }
 
+static gint
+sort_node(gconstpointer a, gconstpointer b)
+{
+    const pcmk_controld_api_node_t *node_a = a;
+    const pcmk_controld_api_node_t *node_b = b;
+
+    return strcmp((node_a->uname? node_a->uname : ""),
+                  (node_b->uname? node_b->uname : ""));
+}
+
```

Comment:
libpe_status has the exhaustively long sort_node_uname.  However, crm_node doesn't currently link against that so I guess we don't want to add the depenency.  On the other hand, that function does seem like something that is more broadly useful.

Reply:
Ah I had forgot about that one ... that is much better. It operates on a different structure, however I can functionize the actual comparison part and put that in libcrmcommon.

Reply:
FYI the reason sort_node_uname() is long is so that we sort nodes like "node8 node9 node10" rather than "node10 node8 node9".